packages/context/README.md

```diff
@@ -0,0 +1,19 @@
+# @textile/context
```

Comment:
I've switching all incoming readmes to use the "pattern". but note, we should decide if @textile/hub should still be the main entry point for everything, or if it should be something like @textile/textile?

Reply:
Good discussion point @andrewxhill

Reply:
what would textile/textile be?

Reply:
{textile}/hub <- textile already is in the namespace. i think all packages should be designed to work with the hub. you can use them in a mode that wont touch the hub, but that's an option the developer chooses. they are still hub packages none-the-less.

Reply:
yup, makes sense. @textile/hub it is!